.github/workflows/ms.desktopvirtualization.scalingplans.yml

```diff
@@ -0,0 +1,135 @@
+name: 'DesktopVirtualization: Scalingplans'
```

Comment:
Please also create a copy in the `.azuredevops/modulePipelines` directory. Just as the GitHub pipelines you can create a simple copy of one of the other pipelines and update the values accordingly